modelz/utils.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+from typing import Literal
+import ssl
+
+
+def strtobool(val: str) -> Literal[0, 1]:
+    # copied from distutils.util cuz importing it is too slow
+    """Convert a string representation of truth to true (1) or false (0).
+
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+    """
+    val = val.lower()
+    if val in ("y", "yes", "t", "true", "on", "1"):
+        return 1
+    elif val in ("n", "no", "f", "false", "off", "0"):
+        return 0
```

Comment:
1. what's the default value?
2. why not return True/False?

Reply:
1. The default value is `ssl_verify = True`.
2. This is actually copied from `distutils.util.strtobool` in the stdlib. I copied the function here because importing this from distutils is slow. The function doesn't return boolean but 0/1 for legacy reasons, but I think it's fine because it doesn't interfere with the purpose of this function.